composer.json

```diff
@@ -10,6 +10,7 @@
         }
     },
     "require": {
+        "php": ">= 8.0.0",
```

Comment:
Is there a reason we should leave 7.x behind for this?

Reply:
I've added `mixed` type-hints where appropriate.

7.x is 1 year EOL.